### PR TITLE
feat: add terms for concessions

### PR DESCRIPTION
**Pull request by bobeal**

### CODE REVIEW

**Review by junjou — APPROVED** [review]

**Review by AHABID — COMMENTED** [review]
I think that its source file  need to be referenced in [smbt-rol.jsonld](https://github.com/easy-global-market/smbt-data-models/blob/feature/add-terms-for-concessions/jsonld-contexts/smbt-rol.jsonld)

**Comment by bobeal** [comment]
Indeed, done.

From: Ahmed Abid ***@***.***>
Date: Friday, 18 March 2022 at 09:27
To: easy-global-market/smbt-data-models ***@***.***>
Cc: Benoit ORIHUELA (EGM) ***@***.***>, Author ***@***.***>
Subject: Re: [easy-global-market/smbt-data-models] feat: add terms for concessions (PR #14)

@AHABID commented on this pull request.

I think that its source file need to be referenced in smbt-rol.jsonld<https://github.com/easy-global-market/smbt-data-models/blob/feature/add-terms-for-concessions/jsonld-contexts/smbt-rol.jsonld>

File: jsonld-contexts/shellfish-table.jsonld [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
+    "@context": {
+        "column": "https://smart-data-models.org/column",
+        "concessionNumber": "https://smart-data-models.org/concessionNumber",
+        "Corporation": "https://schema.org/Corporation",
+        "familyName": "https://schema.org/familyName",
+        "givenName": "https://schema.org/givenName",
+        "ownerNumber": "https://smart-data-models.org/ownerNumber",
+        "parcel": "https://smart-data-models.org/parcel",
```
Comment (AHABID) [inline jsonld-contexts/shellfish-table.jsonld:9]: `parcel` is a property in this case?

**Review by AHABID — APPROVED** [review]

**Comment by bobeal** [comment]
Yes. Estimated translation for « lot », but it can be challenged 😊

From: Ahmed Abid ***@***.***>
Date: Friday, 18 March 2022 at 09:33
To: easy-global-market/smbt-data-models ***@***.***>
Cc: Benoit ORIHUELA (EGM) ***@***.***>, Author ***@***.***>
Subject: Re: [easy-global-market/smbt-data-models] feat: add terms for concessions (PR #14)

@AHABID approved this pull request.

________________________________

In jsonld-contexts/shellfish-table.jsonld<https://github.com/easy-global-market/smbt-data-models/pull/14#discussion_r829787125>:

> @@ -0,0 +1,15 @@

+{

+    ***@***.***": {

+        "column": "https://smart-data-models.org/column",

+        "concessionNumber": "https://smart-data-models.org/concessionNumber",

+        "Corporation": "https://schema.org/Corporation",

+        "familyName": "https://schema.org/familyName",

+        "givenName": "https://schema.org/givenName",

+        "ownerNumber": "https://smart-data-models.org/ownerNumber",

+        "parcel": "https://smart-data-models.org/parcel",

parcel is a property in this case?

**Pull request merged**